internal/builder/known_builder.go

```diff
@@ -11,7 +11,7 @@ type KnownBuilder struct {
 var KnownBuilders = []KnownBuilder{
 	{
 		Vendor:             "Google",
-		Image:              "gcr.io/buildpacks/builder:v1",
+		Image:              "gcr.io/buildpacks/builder:google-22",
 		DefaultDescription: "GCP Builder for all runtimes",
```

Comment:
I think we should also sync the description here with the description in the builders upstream manifest:
https://github.com/GoogleCloudPlatform/buildpacks/blob/ccee07f7459d6e1a5efe15258f99d7c09e2fcae5/builders/gcp/base/google.22.builder.toml#L1

```suggestion
		DefaultDescription: "Ubuntu 22.04 base image with buildpacks for .NET, Dart, Go, Java, Node.js, PHP, Python, and Ruby",
```

cc @matthewrobertson and @kritkasahni-google as recent upstream builder committers:
https://github.com/GoogleCloudPlatform/buildpacks/commits/main/builders/gcp/base/google.22.builder.toml

Reply:
Thank you for pointing it out; it looks fine to me as well. 😄